### PR TITLE
Added excluded prefixes client example

### PR DESCRIPTION
**Pull request by Sol-0**
Signed-off-by: Oleg Solodkov <oleg.solodkov@xored.com>

### CODE REVIEW

File: examples/features/exclude-prefixes-client/README.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+Create test namespace:
+```bash
+NAMESPACE=($(kubectl create -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/b4e64d2c715cb540ea58400cd042412e9d67d42e/examples/use-cases/namespace.yaml)[0])
+NAMESPACE=${NAMESPACE:10}
+```
+
+Select node to deploy NSC and NSE:
+```bash
+NODE=($(kubectl get nodes -o go-template='{{range .items}}{{ if not .spec.taints  }}{{index .metadata.labels "kubernetes.io/hostname"}} {{end}}{{end}}')[0])
+```
```
Comment (denis-tingaikin) [inline examples/features/exclude-prefixes-client/README.md:22]: Do we need a concreate node for this example?
```suggestion
```

File: examples/features/exclude-prefixes-client/README.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    networkservicemesh.io: kernel://nsm-service-1/nsm-1,kernel://nsm-service-2/nsm-2
+spec:
+  containers:
+  - name: alpine
+    image: alpine:3.15.0
+    imagePullPolicy: IfNotPresent
+    stdin: true
+    tty: true
+  nodeSelector:
+    kubernetes.io/hostname: ${NODE}
```
Comment (denis-tingaikin) [inline examples/features/exclude-prefixes-client/README.md:62]:
```suggestion
```

File: examples/features/exclude-prefixes-client/README.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+        - name: spire-agent-socket
+          hostPath:
+            path: /run/spire/sockets
+            type: Directory
+        - name: nsm-socket
+          hostPath:
+            path: /var/lib/networkservicemesh
+            type: DirectoryOrCreate
+      nodeSelector:
+        kubernetes.io/hostname: ${NODE}
```
Comment (denis-tingaikin) [inline examples/features/exclude-prefixes-client/README.md:157]:
```suggestion
```

File: examples/features/exclude-prefixes-client/README.md [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
```
Comment (denis-tingaikin) [inline examples/features/exclude-prefixes-client/README.md:226]:
```suggestion
```

Comment (Sol-0) [reply examples/features/exclude-prefixes-client/README.md]: Done

File: examples/features/exclude-prefixes-client/README.md (hunk 3 above)
Comment (Sol-0) [reply examples/features/exclude-prefixes-client/README.md]: Done

File: examples/features/exclude-prefixes-client/README.md (hunk 2 above)
Comment (Sol-0) [reply examples/features/exclude-prefixes-client/README.md]: Done

File: examples/features/exclude-prefixes-client/README.md (hunk 1 above)
Comment (Sol-0) [reply examples/features/exclude-prefixes-client/README.md]: Done

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**